www/components/Layout.tsx

```diff
@@ -1,36 +1,40 @@
 import React, { ReactNode } from "react";
 import { createTheme, WuiProvider } from "@welcome-ui/core";
 import { darkTheme } from "@welcome-ui/themes.dark";
-import { welcomeTheme } from "@welcome-ui/themes.welcome";
 import { Flex } from "@welcome-ui/flex";
 import { Stack } from "@welcome-ui/stack";
 import { Text } from "@welcome-ui/text";
 import { Box } from "@welcome-ui/box";
 import { Pkg } from "../lib/docs";
 import styled from "styled-components";
 
-const Main = styled(Box)`
-  min-height: 100vh;
-`;
-
 const theme = createTheme(darkTheme);
 
-export default function Layout({
-  title,
-  children,
-}: {
+interface Props {
   title: string;
   children: ReactNode;
-}) {
+  sidebar?: ReactNode;
+}
+
+const Sidebar = styled.div`
+  overflow: auto;
+  background-color: #151515;
+  padding-left: ${(props) => props.theme.space.xxl};
```

Comment:
This is the `styled-components` syntax for using values from the theme. WUI internally uses `@xstyled/styled-components`, and I tried to use that for a bit, but didn't get anywhere with it. If we can make that work, it's preferable; the syntax is much cleaner.

Reply:
I started out with `@xstyled/styled-components` but it didn't play well with server side rendering and I couldn't make it work. Looks like next has better support for regular `styled-components`